admin-client/src/pages/domain/Show.svelte

```diff
@@ -63,6 +67,9 @@
         {#if domain.state === 'pending'}
           <button class="usa-button usa-button--big" disabled>Provision</button>
         {/if}
+        {#if domain.state === 'provisioned'}
```

Comment:
hmmm, since these buttons are disabled until the DNS resolves anyways, maybe it doesn't make sense to have them here at all...

Reply:
I've removed the `domain.state` check and the button simply reads "Loading..." until the DNS results show up.